desci-server/test/integration/guest.test.ts

```
@@ -87,6 +87,7 @@ describe('Guest User Tests', () => {
 
   it('should convert a guest user to a NEW regular user', async () => {
     const guestRes = await request(app).post('/v1/auth/guest').send({ dev: 'true' });
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)